src/MockeryTools/PseudoIntegration/PseudoIntegrationTestCase.php

```diff
@@ -441,6 +441,28 @@ protected function expectRequestWithStringResponseFail(
     }
 
 
+    protected function expectFileContentRequest(string $fileUrl, string $fileContent, string $contentType): void
+    {
+        $psrResponse = new PsrResponse(200, ['Content-Type' => $contentType], $fileContent);
+
+        $this->httpClientMock->expects('request')
+            ->with('GET', $fileUrl, Mockery::any())
```

Comment:
[o] Otázka je, jestli to chceme, ale je tu možnost mít parametrizované i `$options` (3. parametr requestu) a mít výchozí hodnotu `null`.
```suggestion
            ->with('GET', $fileUrl, $expectedClientOptions ?: Mockery::any())
```

To by nám umožnilo testovat i konkrétně nastavené timeouty, jako to děláme v `\BE\AdapterSdk\ThirdParty\FileContentReaderTest`.